junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/descriptor/Validatable.java

```diff
@@ -29,4 +32,13 @@ public interface Validatable {
 	 */
 	void validate(DiscoveryIssueReporter reporter);
 
+	/**
+	 * Report and clear the given list of {@link DiscoveryIssue}s using the
+	 * supplied {@link DiscoveryIssueReporter}.
+	 */
+	static void reportAndClear(List<DiscoveryIssue> issues, DiscoveryIssueReporter reporter) {
```

Comment:
Why is this declared as static member of `Validatable` and as a normal member of `DiscoveryIssueReporter`?

Reply:
Do you mean "and _not_ as a ..."?

Reply:
In order to allow calls like:

```java
reporter.reportAndClear(issues);
```

Reply:
I'm not convinced it's generally useful enough to change it from an implementation detail in Jupiter to a Platform API. I'll ponder it...

Reply:
Got it. Good as-is.